src/content/developers/docs/transactions/index.md

```diff
@@ -138,8 +138,10 @@ Once the transaction has been submitted the following happens:
 2. The transaction is then broadcast to the network and included in a pool with lots of other transactions.
 3. A miner must pick your transaction and include it in a block in order to verify the transaction and consider it "successful".
    - You may end up waiting at this stage if the network is busy and miners aren't able to keep up. Miners will always prioritise transactions with higher `GASPRICE` because they get to keep the fees.
-4. Your transaction will also get a block confirmation number. This is the number of blocks created since the block that your transaction was included in. The higher the number, the greater the certainty that the transaction was processed and recognised by the network. This is because sometimes the block your transaction was included in may not have made it into the chain.
-   - The larger the block confirmation number the more immutable the transaction is. So for higher value transactions, more block confirmations may be desired.
+4. Your transaction is associated with a block confirmation number. Note:
+  - The confirmation number increments as new blocks are created. The higher the number, the greater the certainty that the transaction was processed and recognised by the network. 
+  - Recent blocks may be re-organised. This gives the impression that the transaction was reversed, however, the transaction is still valid but may be included in a different block.
+  - The probability of a re-organisation diminishes with every subsequent block mined, i.e. the larger the block confirmation number, the more immutable the transaction is. So, for higher value transactions, more block confirmations may be desired.
```

Comment:
```suggestion
Your transaction will receive "confirmations". The number of confirmations is the number of blocks created since the block that included your transaction. The higher the number, the greater the certainty that the network processed and recognised the transaction.
   - Recent blocks may get re-organised, giving the impression the transaction was unsuccessful; however, the transaction may still be valid but included in a different block.
   - The probability of a re-organisation diminishes with every subsequent block mined, i.e. the greater the number of confirmations, the more immutable the transaction is.
```